v3/splitkb/kyria_rev3.json

```diff
@@ -0,0 +1,302 @@
+{
+  "name": "SplitKB Kyria",
+  "vendorId": "0x8D1D",
+  "productId": "0xCF44",
+  "keycodes": [
+    "qmk_lighting"
+  ],
```

Comment:
Specified rgb keycodes but not the accompanying `menus`?

Also see the RGB matrix stuff <https://github.com/qmk/qmk_firmware/blob/master/keyboards/splitkb/kyria/rev3/info.json#L125>